From e49e3c0d51eedfd1bb17edc50776631dc88e3722 Mon Sep 17 00:00:00 2001
From: Chiawen Lee <chiawen.lee@mediatek.com>
Date: Wed, 18 Mar 2015 10:14:39 +0800
Subject: [PATCH 3/3] Add Mediatek proprietary formats

Add DRM_FORMAT_MT12 and DRM_FORMAT_MT21 for Mediatek Properity
video block format

Signed-off-by: Chiawen Lee <chiawen.lee@mediatek.com>
---
 include/drm/drm_fourcc.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/drm/drm_fourcc.h b/include/drm/drm_fourcc.h
index 85facb0..fb4d3bb 100644
--- a/include/drm/drm_fourcc.h
+++ b/include/drm/drm_fourcc.h
@@ -107,6 +107,9 @@
 #define DRM_FORMAT_NV16		fourcc_code('N', 'V', '1', '6') /* 2x1 subsampled Cr:Cb plane */
 #define DRM_FORMAT_NV61		fourcc_code('N', 'V', '6', '1') /* 2x1 subsampled Cb:Cr plane */
 
+#define DRM_FORMAT_MT12		fourcc_code('M', 'T', '1', '2') /* Mediatek Block Mode */
+#define DRM_FORMAT_MT21		fourcc_code('M', 'T', '2', '1') /* Mediatek Block Mode */
+
 /*
  * 3 plane YCbCr
  * index 0: Y plane, [7:0] Y
-- 
1.9.1

